lib/dox/formatters/plain.rb

```diff
@@ -2,7 +2,26 @@ module Dox
   module Formatters
     class Plain < Dox::Formatters::Base
       def format
-        body
+        normalize(body)
+      end
+
+      private
+
+      def normalize(data)
+        case data
+        when String
+          to_utf8(data)
+        else
+          data
+        end
+      end
+
+      def to_utf8(stream)
```

Comment:
Add a test for this case.

Reply:
there are no fine-grained specs at all at the moment. Should I write unit tests perhaps?

Reply:
Yes.